pydm/widgets/archiver_time_plot.py

```diff
@@ -548,3 +877,19 @@ def addYChannel(
             useArchiveData=useArchiveData,
             liveData=liveData,
         )
+
+    def replaceToArchivePlot(self, address: str, **kwargs) -> ArchivePlotCurveItem:
+        # This is specifically in order to create an ArchivePlotCurveItem
+        # without changing axes or appending to the row.
+        ArchiveCurve = ArchivePlotCurveItem(**kwargs)
+        [ch.disconnect() for ch in ArchiveCurve.channels() if ch]
+        ArchiveCurve.address = address
+        [ch.connect() for ch in ArchiveCurve.channels() if ch]
+        return ArchiveCurve
```

Comment:
I'm not sure I fully understand what the intent for this method is, but doesn't seem to be called anywhere. Maybe can be removed?

Reply:
It was bad form - it was called from the ArchiveViewer side, but I have now removed it and everything functions as I want it to. Thank you for the note!